SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/model/Question.java

```diff
@@ -12,7 +12,7 @@ public class Question {
     private String fourthAnswer;
     private int correctIndex;
     private Category category;
-    private boolean needsReview;
+    private int needsReview;
```

Comment:
Maybe good to add a short little comment to explain the states that needs review may have: Like what does a 2 mean or any other number for that matter